protocol/database-scripting/README.md

```diff
@@ -10,41 +10,54 @@ Develop the scripting story similarly to any other story. In this case, this wou
 entail running your script at the rails console of your local server.
 
 While some scripts only lend themselves to being performed via pure SQL, where it
-is possible, you should leverage ActiveRecord to perform our INSERT/UPDATE/DELETE
+is possible, you should leverage ActiveRecord to perform our `INSERT`/`UPDATE`/`DELETE`
 operations so that we leverage ActiveRecord validations and callbacks.
 
 While developing the script, it is critical to plan our remediation strategy if
 something does go awry.
 
-One useful approach to consider is to wrap the INSERT/UPDATE/DELETE commands within
+One useful approach to consider is to wrap the `INSERT`/`UPDATE`/`DELETE` commands within
 a transaction, and then to assert on certain states of the database after the script
 is complete, rolling back all changes if the assertions fail.
 
 Further, it's important to be able to execute a complete reversal of your changes
-in the event that there are unexpected failures. Ideally, we would write a reversal
-script to immediately reverse our changes if necessary.
+in the event that there are unexpected failures. Ideally, we would write the commands
+we would need to immediately reverse our changes if necessary (i.e. a reversal script).
 
-For INSERT operations, we would need to retain data about what records ended up
-being INSERT-ed by our script, and then the reversal script would delete those
+For `INSERT` operations, we would need to retain data about what records ended up
+being `INSERT`-ed by our script, and then the reversal script would delete those
 records.
 
-For UPDATE/DELETE operations, we would want to dump the data of all the records
-we will be interacting with into a Temp Table (or Tables). Our reversal script would then
-perform a COPY of the data from the Temp Table into the original table (and a deletion
-of updated records in the case of an Update).
+For `UPDATE`/`DELETE` operations, we would want to dump the data of all the records
+we will be interacting with into a "rescue table" (a table of the which we would keep
+for only around a week before deleting. Such a table would ideally live in a
+`rescue_table.*` DB schema). Our reversal script would then perform a COPY of the data
+from the rescue table into the original table (and a deletion of updated records in the
+case of an Update). Although an inferior option, it alternatively often suffices to dump
+the data into a csv in lieu of a rescue table.
+
+**Handling PHI**
+
+To remain HIPAA compliant, when our scripts touch data that is PHI, it is critically
```

Comment:
Yes, this seems correct.

It's not the format of a file that could be "bad" from a security perspective, it's the location of the information. If the CSV lives on Aptible, that's fine. However, chances that if there's a CSV, it'll be emailed/SSHed around, so better to exclude it in the formal documentation.